python/tests/test_transforms.py

```diff
@@ -0,0 +1,526 @@
+def test_null_human_string(time_transform_name):
+    assert (
+        getattr(transforms, time_transform_name)(TimestamptzType).to_human_string(None)
+        == "null"
+    )
+
+
+@pytest.mark.parametrize(
+    "type_var,value,expected",
+    [
+        (LongType, None, "null"),
+        (DateType, 17501, "2017-12-01"),
+        (TimeType, 36775038194, "10:12:55.038194"),
+        (TimestamptzType, 1512151975038194, "2017-12-01T18:12:55.038194Z"),
+        (TimestampType, 1512151975038194, "2017-12-01T18:12:55.038194"),
+        (LongType, -1234567890000, "-1234567890000"),
+        (StringType, "a/b/c=d", "a/b/c=d"),
+        (DecimalType(9, 2), Decimal("-1.50"), "-1.50"),
+        (FixedType(100), b"foo", "Zm9v"),
+    ],
+)
+def test_identity_human_string(type_var, value, expected):
+    identity = transforms.identity(type_var)
+    assert identity.to_human_string(value) == expected
+
+
+@pytest.mark.parametrize("type_var", [IntegerType, LongType])
+@pytest.mark.parametrize(
+    "input_var,expected",
+    [(1, 0), (5, 0), (9, 0), (10, 10), (11, 10), (-1, -10), (-10, -10), (-12, -20)],
+)
+def test_truncate_integer(type_var, input_var, expected):
+    trunc = transforms.truncate(type_var, 10)
+    assert trunc.apply(input_var) == expected
+
+
+@pytest.mark.parametrize(
+    "input_var,expected",
+    [
+        (Decimal(12.34).quantize(Decimal(".01")), Decimal("12.30")),
+        (Decimal(12.30).quantize(Decimal(".01")), Decimal("12.30")),
+        (Decimal(12.20).quantize(Decimal(".01")), Decimal("12.20")),
+        (Decimal(0.05).quantize(Decimal(".01")), Decimal("0.00")),
+        (Decimal(-0.05).quantize(Decimal(".01")), Decimal("-0.10")),
+    ],
+)
+def test_truncate_decimal(input_var, expected):
+    trunc = transforms.truncate(DecimalType(9, 2), 10)
+    assert trunc.apply(input_var) == expected
+
+
+@pytest.mark.parametrize("input_var,expected", [("abcdefg", "abcde"), ("abc", "abc")])
+def test_truncate_string(input_var, expected):
+    trunc = transforms.truncate(StringType, 5)
+    assert trunc.apply(input_var) == expected
+
+
+@pytest.mark.parametrize(
+    "type_var",
+    [
+        BinaryType,
+        DateType,
+        DecimalType(8, 5),
+        DoubleType,
+        FixedType(8),
+        FloatType,
+        IntegerType,
+        LongType,
+        StringType,
+        TimestampType,
+        TimestamptzType,
+        TimeType,
+        UUIDType,
+    ],
+)
+def test_bucket_method(type_var):
+    bucket_transform = transforms.bucket(type_var, 8)
+    assert str(bucket_transform) == str(eval(repr(bucket_transform)))
+    assert bucket_transform.can_transform(type_var)
+    assert bucket_transform.result_type(type_var) == IntegerType
+    assert bucket_transform.num_buckets == 8
+    assert bucket_transform.apply(None) is None
+    assert bucket_transform.to_human_string("test") == "test"
+
+
+@pytest.mark.parametrize(
+    "type_var,value,expected",
```

Comment:
Expected is misleading for a `test_truncate_method` test. It should be more clear that it is the expected representation of the argument as a human-readable string.

Reply:
Updated the arg name and also added the expected values for apply.